assets/app/features/upgrade/controllers/upgrade-landing.controller.spec.js

```diff
@@ -79,6 +211,69 @@ describe('Upgrade Landing Controller', function() {
         should.exist(controller);
     });
 
+    describe('when created while prepare is not running', function() {
```

Comment:
No. "before running prepare nodes" doesn't express what is being tested here. The key is that we want to check the controller creation/activation (i.e. page load) in different cases. The "describe" parts are destined to sound weird sooner or later when you start nesting things.